[2025-05-07T00:40:31.136+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-07T00:40:31.179+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews.normalize_and_clean_data manual__2025-05-06T20:00:25.137792+00:00 [queued]>
[2025-05-07T00:40:31.202+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews.normalize_and_clean_data manual__2025-05-06T20:00:25.137792+00:00 [queued]>
[2025-05-07T00:40:31.221+0000] {taskinstance.py:2867} INFO - Starting attempt 18 of 19
[2025-05-07T00:40:31.249+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): normalize_and_clean_data> on 2025-05-06 20:00:25.137792+00:00
[2025-05-07T00:40:31.268+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=11144) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-07T00:40:31.273+0000] {standard_task_runner.py:72} INFO - Started process 11159 to run task
[2025-05-07T00:40:31.272+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews', 'normalize_and_clean_data', 'manual__2025-05-06T20:00:25.137792+00:00', '--job-id', '693', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmp0su4j0b5']
[2025-05-07T00:40:31.274+0000] {standard_task_runner.py:105} INFO - Job 693: Subtask normalize_and_clean_data
[2025-05-07T00:40:31.381+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews.normalize_and_clean_data manual__2025-05-06T20:00:25.137792+00:00 [running]> on host 8280c28b7f0f
[2025-05-07T00:40:31.655+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews' AIRFLOW_CTX_TASK_ID='normalize_and_clean_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-06T20:00:25.137792+00:00' AIRFLOW_CTX_TRY_NUMBER='18' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-06T20:00:25.137792+00:00'
[2025-05-07T00:40:31.659+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-07T00:40:31.661+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-07T00:40:31.662+0000] {logging_mixin.py:190} INFO - Current task name:normalize_and_clean_data state:running start_date:2025-05-07 00:40:31.183566+00:00
[2025-05-07T00:40:31.664+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews and current dag run status:running
[2025-05-07T00:40:31.665+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-07T00:40:31.685+0000] {base.py:84} INFO - Retrieving connection 'postgres_defaut'
[2025-05-07T00:40:31.706+0000] {warnings.py:112} WARNING - /opt/***/dags/storeData.py:381: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql("SELECT * FROM review.raw_reviews", conn)

[2025-05-07T00:40:33.176+0000] {warnings.py:112} WARNING - /opt/***/dags/storeData.py:414: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql_query(query, conn)

[2025-05-07T00:40:33.251+0000] {logging_mixin.py:190} INFO - Data normalized and cleaned successfully.
[2025-05-07T00:40:33.254+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-07T00:40:33.272+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-07T00:40:33.273+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews, task_id=normalize_and_clean_data, run_id=manual__2025-05-06T20:00:25.137792+00:00, execution_date=20250506T200025, start_date=20250507T004031, end_date=20250507T004033
[2025-05-07T00:40:33.311+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-07T00:40:33.312+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-07T00:40:33.313+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews queued_at:2025-05-07 00:37:43.303466+00:00
[2025-05-07T00:40:33.314+0000] {logging_mixin.py:190} INFO - Task hostname:8280c28b7f0f operator:PythonOperator
[2025-05-07T00:40:33.345+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-07T00:40:33.403+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-07T00:40:33.406+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
